roadmap/implementers-guide/src/types/disputes.md

```diff
@@ -0,0 +1,72 @@
+# Disputes
+
+## DisputeStatementSet 
+
+```rust
+/// A set of statements about a specific candidate.
+struct DisputeStatementSet {
+    candidate_hash: CandidateHash,
+    session: SessionIndex,
+    statements: Vec<(DisputeStatement, ValidatorIndex, ValidatorSignature)>,
```

Comment:
Could we not use `SignedStatement` types for this?

Reply:
No, they're different. And `Signed<T>` is unwieldy for the reconstruction of statements.